From 6073ac46e693ebd0ce904cda4aea2159b6e18efb Mon Sep 17 00:00:00 2001
From: Steve deRosier <steve.derosier@lairdtech.com>
Date: Wed, 29 Jun 2016 16:32:18 -0700
Subject: [PATCH] laird: wb45n: dt: remove dma from mmc0 to avoid bug

This change removes the DMA via the device tree for the mmc0 bus. This
avoids a scatter-write error -110 in the ath6kl_sdio driver. It is a
workarround and may be removed when we fix the bug in the driver.

Bug: 8612
---
 arch/arm/boot/dts/wb45n.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/wb45n.dts b/arch/arm/boot/dts/wb45n.dts
index 571c0525e5e8..2d1fe0c0830a 100644
--- a/arch/arm/boot/dts/wb45n.dts
+++ b/arch/arm/boot/dts/wb45n.dts
@@ -89,6 +89,8 @@
 					&pinctrl_mmc0_slot0_clk_cmd_dat0
 					&pinctrl_mmc0_slot0_dat1_3>;
 				status = "okay";
+				/delete-property/dmas;
+				/delete-property/dma-names;
 				slot@0 {
 					reg = <0>;
 					bus-width = <4>;
-- 
2.11.0

